modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -1,13 +1,14 @@
 // Module included in the following assemblies:
 //
 // * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
-
+:product-version: 4.9
```

Comment:
Was this used for testing and should be removed?

Reply:
Yes. Conditional formatting doesn't work without it do to hard coding in asciibinder, until that gets fixed. Removed.